cli/functionary/package.py

```diff
@@ -26,6 +29,10 @@ def generateYaml(output_dir: str, name: str, language: str):
         path.write_text(yaml.dump(metadata))
 
 
+def getEnvironmentID():
```

Comment:
This should be `get_environment_id`.  I realize that the function immediately before this fails to follow the correct [function naming convention](https://peps.python.org/pep-0008/#function-and-variable-names) as well, which is on me since I reviewed that code too.  But let's fix this one (and `environmentId` below) and work toward some consistency.